Source/WebCore/platform/graphics/mac/controls/WebControlView.mm

```diff
@@ -176,4 +176,16 @@ - (CFDictionaryRef)_coreUIDrawOptionsWithFrame:(NSRect)cellFrame inView:(NSView
 
 @end
 
+@implementation WebSwitchView
+
+- (id) init
+{
+    _nativeSwitch = adoptNS([[NSSwitch alloc] init]);
+    [_nativeSwitch setWantsLayer:YES];
+    [_nativeSwitch setControlSize:NSControlSizeRegular];
```

Comment:
We should support all 4  `NSControlSizes` similar to checkboxes and radio buttons.